### PR TITLE
add tidying branch

### PR DESCRIPTION
**Pull request by tealiie**

### CODE REVIEW

**Comment by tealiie** [comment]
Cleaning up indentation

**Comment by rene-justin-dado** [comment]
Hi, @tealiie. Just looked through changes. When you get the time, could you also clean up the li tags so the opening and closing tags are tabbed on the same level. :D
(This is part of the assignment btw. Not just trying to make your life harder. XD)

**Comment by tealiie** [comment]
@rene-justin-dado to clarify, does that mean the <> and </> are indented on the same line?

**Comment by rene-justin-dado** [comment]
Yes, please. :)

**Comment by tealiie** [comment]
@rene-justin-dado all tidied, please have a look

File: pristine-zen-garden.html [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+              <li> <a href="/201/" class="design-name">Lily Pond</a> by <a href="#" class="designer-name">Rose Thorogood</a>
+              </li>    
+
+              <li> <a href="/200/" class="design-name">Icicle Outback</a> by           <a href="http://www.timovirtanen.com/" class="designer-name">Timo Virtanen</a>
+              </li>         
+
+              <li> <a href="/199/" class="design-name">CSS ZEN ARMY</a> by <a href="http://www.niceguy.com" class="designer-name">Carl Desmond</a>
+              </li>   
+
+              <li><a href="/198/" class="design-name">The Original</a> by           <a href="#" class="designer-name">Joachim Shotter</a>
```
Comment (rene-justin-dado) [inline pristine-zen-garden.html]: spaces after by

File: pristine-zen-garden.html [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+              <li><a href="/203/" class="design-name">Tiny Blue</a> by <a href="http://www.timovirtanen.com/" class="designer-name">Timo Virtanen</a>
+              </li>  
+
+              <li> <a href="/202/" class="design-name">Retro Theater</a> by <a href="#" class="designer-name">Eric Rogé</a>
+              </li>   
+
+              <li> <a href="/201/" class="design-name">Lily Pond</a> by <a href="#" class="designer-name">Rose Thorogood</a>
+              </li>    
+
+              <li> <a href="/200/" class="design-name">Icicle Outback</a> by           <a href="http://www.timovirtanen.com/" class="designer-name">Timo Virtanen</a>
```
Comment (rene-justin-dado) [inline pristine-zen-garden.html]: spaces after by

**Comment by rene-justin-dado** [comment]
li's look good.
Sorry, just a couple more lines. (95 and 101)
Just delete the spaces after the words 'by' please. Lemme know when I can look at it again. :)

**Comment by tealiie** [comment]
@rene-justin-dado you can have a look now

**Comment by rene-justin-dado** [comment]
All good, @tealiie! I'll just merge the pull then i'll delete the pull request. :)

**Pull request merged**